### PR TITLE
[feat]curvefs: add bs create volume snapshot

### PR DESCRIPTION
**Pull request by ApiaoSamaa**
### What problem does this PR solve?

Issue Number: #2585  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
+/*
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/create/volume/snapshot/snapshot.go:1]: add License

**Review by Cyber-SiKu — COMMENTED** [review]
Maybe the problem is not clearly described, what I want to achieve is this:
https://github.com/opencurve/curve/blob/6a7415134548286998c28f486e48e685d397daa0/tools/snaptool/curltool.py#L85-L95

**Comment by Cyber-SiKu** [comment]
> Maybe the problem is not clearly described, what I want to achieve is this:
> 
> https://github.com/opencurve/curve/blob/6a7415134548286998c28f486e48e685d397daa0/tools/snaptool/curltool.py#L85-L95

There should be a problem with your implementation. What you want to achieve is this, which should be to create an http request.

**Comment by ApiaoSamaa** [comment]
Hi! I found the problem, which was caused due to my being misled into calling the wrong interface, and now two solutions come into my mind: 1. The client contracts an http packet. 2. Add a new roc interface. If you want a uniform format, we may use solution 2, but 1 is easier to achieve. @Cyber-SiKu

**Comment by Cyber-SiKu** [comment]
> Hi! I found the problem, which was caused due to my being misled into calling the wrong interface, and now two solutions come into my mind: 1. The client contracts an http packet. 2. Add a new roc interface. If you want a uniform format, we may use solution 2, but 1 is easier to achieve. @Cyber-SiKu

Option 1 is just fine, you can refer to this [pr](https://github.com/opencurve/curve/pull/2713)

**Comment by Cyber-SiKu** [comment]
@ApiaoSamaa Did you encounter any problems?

**Comment by Cyber-SiKu** [comment]
pls fix dco && conflicts

**Comment by SeanHai** [comment]
Do you want to continue this issue?

**Comment by ApiaoSamaa** [comment]
> Do you want to continue this issue?↳

Sorry for my delay... I might quit TuT

**Comment by wuhongsong** [comment]
pr stale,you can reopen it later.

**Pull request closed without merging**